### PR TITLE
Add EDA Data & Mapping Files

### PR DESCRIPTION
**Pull request by dgamboni**
We're updating our internal tooling to include an EDA Data file (.sql) and an EDA mapping file (release org shape, regression_org flow) for use by QA. These will be located in a newly created datasets/qa folder.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Comment by dgamboni** [comment]
@lifewithryan @jofsky I am interested in keeping a mapping.yml file and an EDA data file in the repo so anyone testing EDA can use the data in their scratch orgs if they like. I pinged you offline to set up a time to review the process. Thanks!

**Comment by robertbwatson** [comment]
@dgamboni @jofsky @lifewithryan @MatthewBlanski The test data set that Omega has been working on should be wrapping up this week. We had intended to share it with the EDA team when we were finished.

From my brief glance through this PR, our data sets don't conflict too much -- I believe we have more contacts, as well as program enrollments, affiliations, plan requirements, and applications. Much of the data was sourced from the Pathways data set and data from the CCE team.

How would you like to move forward with this? Would you still find it valuable to have our data set as well, or perhaps we can work on combining them together in EDA? Our data set will eventually evolve over time to be more applicant-centric, but our starting point might be good to share.

**Comment by dgamboni** [comment]
> @dgamboni @jofsky @lifewithryan @MatthewBlanski The test data set that Omega has been working on should be wrapping up this week. We had intended to share it with the EDA team when we were finished.
> 
> From my brief glance through this PR, our data sets don't conflict too much -- I believe we have more contacts, as well as program enrollments, affiliations, and applications. Much of the data was sourced from the Pathways data set and data from the CCE team.
> 
> How would you like to move forward with this? Would you still find it valuable to have our data set as well, or perhaps we can work on combining them together in EDA? Our data set will eventually evolve over time, but our starting point might be good to share.

@robertbwatson I was using this particular data set as a personal test to see what the process is (ie, to store it in github, then bring it back down and make local changes, then  re-upload it) and also to have a small, familiar dataset for testing. The CCE dataset was more complete, and also better for any customer-facing use; I had trouble loading it back into scratch orgs without turning off triggers, though, so I created this simpler QE dataset. 

I'd think it would be great to have different datasets for different purposes if possible. Could we use a dataset naming convention perhaps?

**Comment by robertbwatson** [comment]
> @robertbwatson I was using this particular data set as a personal test to see what the process is (ie, to store it in github, then bring it back down and make local changes, then re-upload it) and also to have a small, familiar dataset for testing. The CCE dataset was more complete, and also better for any customer-facing use; I had trouble loading it back into scratch orgs without turning off triggers, though, so I created this simpler QE dataset.
> 
> I'd think it would be great to have different datasets for different purposes if possible. Could we use a dataset naming convention perhaps?

@dgamboni Thanks for the additional context and I agree, having different datasets for different use cases might be ideal. I look forward to sharing out our test data with you all soon!

File: DsEdaData.sql [hunk 1]
```diff
@@ -0,0 +1,761 @@
+BEGIN TRANSACTION;
+CREATE TABLE "Account" (
+	sf_id VARCHAR(255) NOT NULL, 
+	"Name" VARCHAR(255), 
+	"RecordTypeId" VARCHAR(255),
```
Comment (jofsky) [inline DsEdaData.sql]: Any reason for "" @dgamboni just wondering :)

Comment (dgamboni) [reply DsEdaData.sql]: @jofsky I used the task `cci task run extract_dataset` which automatically created the file. I uploaded it without edits.

**Comment by dgamboni** [comment]
@lifewithryan Would you mind reviewing this PR? 

@jofsky reviewed with me, and we decided to create a datasets folder within EDA repo to hold varying different datasets & mapping files that could be used by different groups or projects. For now we've created a datasets/qa folder, with 2 files in it. Others could create their own folders within datasets as well. Thanks for your input!

**Pull request merged**